src/templates/export-menu.html

```
@@ -120,14 +120,28 @@ <h5 class='modal-title' id='qrpngExportModalLabel'><i class='fas fa-fw fa-file-a
+    </a>
+    <button type='button' class='dropdown-item' data-action='toggle-modal' data-target='qrpngExportModal'>
+      <i class='fas fa-qrcode fa-fw'></i> {{ 'QR Code'|trans }}
+    </button>
+    <a class='dropdown-item' href='api/v2/{{ Entity.entityType.value }}/{{ Entity.id }}' target='_blank' rel='noopener noreferrer'>
+      <i class='fas fa-code fa-fw'></i> {{ 'JSON'|trans }}
+    </a>
     <hr>
-      {% set isDisabled = App.Config.configArr.dspace_host == '' or Entity.isReadOnly %}
-    <div class='dropdown-item {{ isDisabled ? 'disabled' }}' data-action='open-dspace-modal'><i class='fas fa-file-export fa-fw {{ isDisabled ? 'color-weak' }}'></i> {{ 'Export to DSpace'|trans }}</div>
+    {% set isDisabled = App.Config.configArr.dspace_host == '' or Entity.isReadOnly %}
+    <button type='button' class='dropdown-item {{ isDisabled ? 'disabled' }}' data-action='open-dspace-modal'{{ isDisabled ? 'disabled aria-disabled="true"' : '' }}>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)